src/main/java/org/snomed/snowstorm/validation/ConceptValidationHelper.java

```diff
@@ -23,42 +18,29 @@
 	private ConceptValidationHelper() {
 	}
 
-	public static List<InvalidContent> validate(final Concept concept, final String branchPath, final DroolsValidationService validationService,
-			final ObjectMapper objectMapper) throws JsonProcessingException, ServiceException {
+	public static InvalidContentWithSeverityStatus validate(final Concept concept, final String branchPath, final DroolsValidationService validationService) throws ServiceException {
 		final List<InvalidContent> invalidContents = validationService.validateConcept(branchPath, generateUUIDSIfNotSet(concept));
 		final List<InvalidContent> invalidContentErrors = invalidContents.stream().filter(invalidContent -> invalidContent.getSeverity() == Severity.ERROR).collect(Collectors.toList());
-		if (invalidContentErrors.isEmpty()) {
-			return invalidContents.stream().filter(invalidContent -> invalidContent.getSeverity() == Severity.WARNING).collect(Collectors.toList());
-		}
-		throw new DroolsValidationException(objectMapper.writeValueAsString(invalidContentErrors).replace("\n", ""));
+		return invalidContentErrors.isEmpty() ? new InvalidContentWithSeverityStatus(invalidContents.stream().filter(invalidContent ->
```

Comment:
The filter probably isn't required here? Because if there are no errors we can assume the list only contains warnings.

Reply:
Nice spot, fixed.